doc/content/gateways/lairdsentrius/_index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Laird Sentrius™ RG1xx LoRaWAN Gateway"
+description: ""
+---
+
+The **Laird Sentrius™ RG1xx** is a secure, scalable, robust LoRaWAN-Enabled Gateway that supports Wi-Fi, Ethernet, and LTE backends.
+
+This page contains information about connecting the Laird Sentrius™ RG1xx to {{% tts %}}
+
+<!--more-->
+
+The technical specifications can be found in [Laird's official documentation](https://www.lairdconnect.com/wireless-modules/lorawan-solutions/sentrius-rg1xx-lorawan-gateway-wi-fi-ethernet-optional-lte-us-only#documentation). The Laird Sentrius™ RG1xx LoRaWAN Gateway supports two ways to connect with {{% tts %}}, using either the Semtech Packet Router or LoRa Basic Station.
```

Comment:
```suggestion
The technical specifications can be found in [Laird's official documentation](https://www.lairdconnect.com/wireless-modules/lorawan-solutions/sentrius-rg1xx-lorawan-gateway-wi-fi-ethernet-optional-lte-us-only#documentation). The Laird Sentrius RG1xx LoRaWAN Gateway supports two ways to connect with {{% tts %}}, using either the Semtech Packet Router or LoRa Basic Station.
```

Only use TM or R at the top of the page. You can leave it out at all other occurrences when you use the Trademarked/corpyrighted names.